.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -214,6 +214,25 @@ def has_dg_changes():
     )
 
 
+@functools.cache
+def has_component_integration_changes():
+    """Check for changes in integrations that implement components."""
+    component_integrations = [
+        "dagster-sling",
+        "dagster-dbt",
+        "dagster-databricks",
+        "dagster-airbyte",
+        "dagster-powerbi",
+        "dagster-omni",
+        "dagster-fivetran",
+        "dagster-dlt",
+    ]
```

Comment:
@alangenfeld might not be the best for maintainability. We could do some programmatic searching for component implementations.